src/metrics/migrations/0001_initial.py

```diff
@@ -1,52 +1,17 @@
 # -*- coding: utf-8 -*-
 # Generated by Django 1.11.10 on 2018-05-14 03:58
 from __future__ import unicode_literals
-
-from django.db import migrations, models, connection
-from django.db.utils import OperationalError, ProgrammingError
+from django.db import migrations, models
 import django.db.models.deletion
 from core import settings
 from pathlib import Path
 import logging
 
 LOG = logging.getLogger(__name__)
 
-#
-# piggybacking off of migrations because this update is awkward.
-# temporary, remove once all databases updated
-#
-
-FLAG = Path(settings.PROJECT_DIR, ".migrations-table-updated.flag")
-if not FLAG.exists():
-    try:
-        try:
-            # case: pre-existing database
-            LOG.info("flag %s not found, updating django_migrations.app values from 'metrics' to 'article_metrics'", FLAG)
-            cursor = connection.cursor()
-            sql = "update django_migrations set app='article_metrics' where app = 'metrics'"
-            cursor.execute(sql)
-            FLAG.touch()
-        except (OperationalError, ProgrammingError) as err:
-            msg = str(err).splitlines()[0]
-
-            # case: brand new database, no old data to migrate
-            sqlite = 'no such table: django_migrations'
-            psql = 'relation "django_migrations" does not exist'
-            if msg in [sqlite, psql]:
-                # no migrations? no problems
-                FLAG.touch()
-
-            # case: unhandled
-            else:
-                raise
-
-    except BaseException:
-        LOG.error("unhandled exception attempting to update django_migrations.app values")
-        raise
-
-#
-#
-#
+# lsh 2019-03-27: delete once the flag is absent
+OLD_FLAG = Path(settings.PROJECT_DIR, ".migrations-table-updated.flag")
+OLD_FLAG.exists() and OLD_FLAG.unlink()
```

Comment:
May be more clear with an `if`

Reply:
just as clear. it will be gone on Friday regardless

Reply:
I agree it doesn't really matter in this case (and approved), but in general what is clear/understandable will be up to the reader of this code :sweat_smile: it's like saying "I don't understand matrix calculus"/"it's just numbers"

Reply:
again, I disagree. the above is perfectly fine. it's *not* like 

>  "I don't understand matrix calculus"/"it's just numbers"

it's more like 

> "I don't understand boolean logic", "but you're a programmer!!!"